src/results/financial.jl

```diff
@@ -53,6 +53,8 @@
 - `om_and_replacement_present_cost_after_tax` Present value of all O&M and replacement costs, after tax.
 - `developer_om_and_replacement_present_cost_after_tax` Present value of all O&M and replacement costs incurred by developer, after tax.
 - `offgrid_microgrid_lcoe_dollars_per_kwh` Levelized cost of electricity for modeled off-grid system.
+- `lifecycle_emissions_cost_climate` LCC component if Settings input include_climate_in_objective is true. Present value of CO2 emissions cost over the analysis period.
```

Comment:
Should these docstrings be updated to maybe say 'Present value of CO2 emissions cost over the analysis period. Minimized LCC component if Settings input include_climate_in_objective is true' since we minimize climate and health costs depending on include_climate_in_objective and include_health_in_objective?

It seems like we are always adding these two fields to the model.

Reply:
Second proposal: 
"Present value of CO2 emissions cost over the analysis period. LCC component (i.e., included in LCC) only if Settings input include_climate_in_objective is true."